### ISSUE TITLE
Deleting a project does not always work

### ISSUE THREAD

**Issue by gideonthomas** [mozilla/publish.webmaker.org#132]
I think this problem is related to the issue where some of us delete projects on thimble, then refresh the page and see it back again.

I recently saw some logs in my postgres session that might help with this:
```
ERROR:  update or delete on table "files" violates foreign key constraint "publishedfiles_file_id_foreign" on table "publishedFiles"
DETAIL:  Key (id)=(26) is still referenced from table "publishedFiles".
STATEMENT:  delete from "projects" where "id" = $1
```

**Comment by sedge** [comment]
After reseeding my database I'm having trouble reproducing this. Can you try re-running `npm knex` and see if you run into this again?

**Comment by sedge** [comment]
This was closed in #141
